### ISSUE TITLE
Titlebar hover messages not translated but...

### ISSUE THREAD

**Issue by henrikfalk** [obsidianmd/obsidian-translations#423]
The following hover messages in the titlebar are not translated when I use "Dansk" (Danish) language:
- "Go back"
- "Go forward"
- "Minimize"
- "Maximize"
- "Close window"

One strange thing though. When I open a vault "Maximize" is in English and if I resize the window it changes to "Maksimer" in Danish. But the rest is still in English of a resize.

When vault opened:
![Screenshot_20210407_131744](https://user-images.githubusercontent.com/30142914/113858325-e0af9980-97a3-11eb-8677-c6ab77af966f.png)
When vault window resized:
![Screenshot_20210407_131218](https://user-images.githubusercontent.com/30142914/113858320-e0170300-97a3-11eb-8057-c4f3e8cb574a.png)

Here are some other screenshots showing that even after a resize it is not translated:
![Screenshot_20210407_124640](https://user-images.githubusercontent.com/30142914/113858554-2704f880-97a4-11eb-86ac-d81e4c456435.png)
![Screenshot_20210407_124650](https://user-images.githubusercontent.com/30142914/113858556-2704f880-97a4-11eb-8ef7-e0f3a06cd467.png)
![Screenshot_20210407_124615](https://user-images.githubusercontent.com/30142914/113858551-266c6200-97a4-11eb-8e98-d58f1a47df75.png)

I think the source is (from "da.json"):
```
"window": {
			"maximize": "Maksimer",
			"minimize": "Minimer",
			"restore-down": "Gendan fra maksimeret",
			"close-window": "Luk vindue",
			"go-back": "Gå tilbage",
			"go-forward": "Gå frem"
		}
```
So apparently one translation "Maximize" works BUT only after you have resized the spp window.

Themes does not seem to have any effect on the issue.

This is my installation info (In Danish but I think you can understand it):
![Screenshot_20210407_132801](https://user-images.githubusercontent.com/30142914/113859469-3a649380-97a5-11eb-80f5-421d1298af36.png)

**Comment by henrikfalk** [comment]
I found that the translation "restore-down": "Gendan fra maksimeret" also works after resize or maximize. It is the same button as "Maximize" so it makes sense.

Another thing. I am almost certain that the translations have been working in some previous versions of Obsidian.

**Comment by henrikfalk** [comment]
Seems to be fixed in 0.12.0
